contracts/contracts/auction/TokenValidatorAuction.sol

```diff
@@ -0,0 +1,49 @@
+pragma solidity ^0.5.8;
+
+import "../lib/Ownable.sol";
+import "./TokenDepositLocker.sol";
+import "../token/IERC20.sol";
+import "./BaseValidatorAuction.sol";
+
+
+contract TokenValidatorAuction is BaseValidatorAuction {
+    IERC20 public auctionnedToken;
```

Comment:
Can we use simply `token` or `bidToken` or so?
auctionnedToken sounds like we are auctioning the token to me.

Reply:
bidToken sounds good